maths/permutations.py

```diff
@@ -0,0 +1,44 @@
+"""
+Reference: https://en.wikipedia.org/wiki/Permutation
+"""
+
+from math import factorial
+
+def permutations(n: int, r: int) -> int:
+    """
+    Returns the number of different ways of r length which can
+    be made from n values where relative order of choice matters 
+    and n >= k.
+
+    Examples:
+    >>>permutations(10, 5)
```

Comment:
```suggestion
    >>> permutations(10, 5)
```
The space is important.

Reply:
I implemented the corrections locally on my system and pushed it to the fork of the project in my account. Can you help me how to update it over here?